src/cms-content/vaccines/index.ts

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react';
+import { getStateCode, Region } from 'common/regions';
+
+export interface StateVaccineData {
+  state: string;
+  generalInformationUrl: string;
+  howToGetVaccinatedUrl: string;
+}
+
+const fetchVaccineData = (region: Region) => {
+  const stateCode = getStateCode(region);
+  // State code required
+  if (!stateCode) {
+    return;
+  }
+  return fetch(`/cms-content/vaccines/${stateCode.toLowerCase()}.json`).then(
+    res => res.json() as Promise<StateVaccineData>,
+  );
```

Comment:
We need to handle the case when the data is not found (maybe return an object with the hidden flag set to true?)